utils/nextflow_minitest/gtftobed.nf

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl=2
+gtf=""
+
+process gtfToBed{
+  container "docker.io/apaeval/gtftobed:1.0"
+  publishDir "gtftobed"
+
+  input:
+  path gtf
+    
+  output:    
+  path "*.bed" , emit: bed
+
+  script:
+  """
+  python3 /app/gtfTobed.py --gtf $gtf
```

Comment:
see comment in README and Dockerfile

Reply:
Now it is executed as below:
https://github.com/iRNA-COSI/APAeval/blob/5a2673ea3b5166174f90f793cba01bd5e511eaff/utils/nextflow_minitest/gtftobed.nf#L20